packages/notifications/push/amplify_push_notifications/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.4.0
+
+- Minor bug fixes and improvements
```

Comment:
shall we add this fix to the change log?
- fix: push notification flush events ([#5215](https://github.com/aws-amplify/amplify-flutter/pull/5215))